src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,40 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private K[] keys = (K[]) new Object[MAX_ARRAY_SIZE];
+    private V[] values = (V[]) new Object[MAX_ARRAY_SIZE];
+    private int sizeOfStorage = 0;
+
     @Override
     public void put(K key, V value) {
+        if (sizeOfStorage >= MAX_ARRAY_SIZE) {
```

Comment:
```suggestion
        if (sizeOfStorage == MAX_ARRAY_SIZE) {
```
storage size will never be more than MAX_ARRAY_SIZE